src/main/java/StringAddCalculator.java

```diff
@@ -0,0 +1,64 @@
+import java.util.Arrays;
+import java.util.List;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class StringAddCalculator {
+    private static String splitDelimiter;
+    private static String target;
+    public static int splitAndSum(String text) throws Exception {
+        splitDelimiter = "[,:]";
+        target = text;
+
+        if(isBlank(target))
+            return 0;
+
+        changeDelimiterAndTargetIfAvailable(text);
+
+        if(isSingleInteger(target))
+            return Integer.parseInt(target);
```

Comment:
[코딩 컨벤션 7. 4 if, if-else, if else-if else 문](https://myeonguni.tistory.com/1596)을 한번 확인해보시면 좋을거 같아요! 😄 
`* if 문은 항상 중괄호를 사용한다`
```suggestion
        if(isSingleInteger(target)) {
            return Integer.parseInt(target);
        }
```

Reply:
@parkeeseul
컨벤션! 다음번엔 꼭 지키도록 하겠습니다!